.github/workflows/build.yaml

```diff
@@ -21,11 +23,13 @@ jobs:
       # as the push and pull operations require the URL of the repository
       # to be in lowercase.
       GITHUB_REPOSITORY_NAME: ${{ steps.repository-name.outputs.lowercase }}
+      GITHUB_REPOSITORY_OWNER: ${{ steps.repository-owner.outputs.lowercase }}
```

Comment:
Also, why the transition to using `github.repository_owner` rather than `github.repository` especially with the container repository URLs?

Reply:
It was neccessary, because with `github.repository` the container would have another level ( [USER|ORG]/[repo]/node-build ), which couldn't be accessed by the package api.
When moved to the toplevel ( [USER|Org]/node-build) it was. 
Not sure if that is problematic. 
Haven't checked if the dynamic removal of the images could be achieved by using the DockerApi of GHCR and preserving the `github.repository` for the container urls.